q2_diversity/_alpha/_visualizer.py

```diff
@@ -16,20 +16,16 @@
 import scipy
 import numpy as np
 import pandas as pd
-import qiime2
 from statsmodels.sandbox.stats.multicomp import multipletests
-import q2templates
 import biom
 import itertools
+
+import qiime2
+import q2templates
```

Comment:
These two can be put back where they came from - the pep8 convention is to group external deps, these were already fine as-is.

Reply:
Thanks for bringing this up. It's something I've been wondering about lately. The PEP8 language is:

>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.

For a platform built across many repos, where some are truly third-party and some are not, it almost feels like there's another class of imports: platform-local but non-package-local (i.e. packages native to the QIIME 2 platform, especially if built by core QIIME 2 devs). These land in a gap between "third party imports" and "local application" imports. 

I'm over-engineering here, and will just put "non-package-local" imports in with the other third-party tools we use in future.